cassandra/src/test/java/org/apache/metamodel/cassandra/CassandraDataContextTest.java

```diff
@@ -113,7 +114,7 @@ public void testSchemaAndSimpleQuery() throws Exception {
 
     @Test
     public void testWhereColumnEqualsValues() throws Exception {
-        DataSet ds = dc.query().from(testTableName).select("id").and("title").where("id").isEquals(firstRowId)
+        DataSet ds = dc.query().from(testTableName).select("id").and("title").where("title").isEquals(firstRowTitle)
```

Comment:
No but the query no longer represented a "filtered dataset" type query because it got optimized. In fact the new test method is identical to the old test method, except that the assertion of it being a `InMemoryDataSet` is there to show the different paths that the two queries take depending on what column is being filtered on.